packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -1,10 +1,11 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { withRouter } from 'react-router-dom';
-import { loginUrl, routes, PlatformContext } from '@deriv/shared';
+import { loginUrl, routes, PlatformContext, isMobile } from '@deriv/shared';
 import { getLanguage } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import { WS } from 'Services';
+import { RudderStack } from '@deriv/analytics';
```

Comment:
@amir-deriv you removed @deriv/analytic but you are importing this rudderstack from that deleted package :thinking: